shared/src/vyxal/ListHelpers.scala

```diff
@@ -670,6 +670,11 @@ object ListHelpers:
   def gradeUp(iterable: VAny)(using Context): VList =
     VList.from(makeIterable(iterable).zipWithIndex.sortBy(_._1).map(_._2))
 
+  def gradeDown(iterable: VAny)(using Context): VList =
+    VList.from(
+      makeIterable(iterable).zipWithIndex.sortBy(_._1).reverse.map(_._2)
```

Comment:
Iirc there's some cases where this doesn't actually return the list graded down.

Same reason I had to change the version 2 implementation from this method. It didn't match up with what apl and jelly produced.

Reply:
Oh yeah, when the list isn't unique.

Reply:
Can you or @ysthakur fix it? I'm not sure how to.

Reply:
@Lyxal I'm pretty sure this *is* the same method as v2. I tested it with non-unique lists, and it gives the same results as v2's `⇩`.